content/en/docs/tasks/debug-application-cluster/crictl.md

```diff
@@ -236,98 +246,103 @@ Using `crictl` to run a pod sandbox is useful for debugging container runtimes.
 On a running Kubernetes cluster, the sandbox will eventually be stopped and
 deleted by the Kubelet.
 
-1.  Create a JSON file like the following:
+1. Create a JSON file like the following:
 
-      ```json
-      {
-          "metadata": {
-              "name": "nginx-sandbox",
-              "namespace": "default",
-              "attempt": 1,
-              "uid": "hdishd83djaidwnduwk28bcsb"
-          },
-          "logDirectory": "/tmp",
-          "linux": {
-          }
-      }
-      ```
+   ```json
+   {
+     "metadata": {
+       "name": "nginx-sandbox",
+       "namespace": "default",
+       "attempt": 1,
+       "uid": "hdishd83djaidwnduwk28bcsb"
+     },
+       "logDirectory": "/tmp",
```

Comment:
nit: `logDirectory` and `linux` fields should be aligned with `metadata`?

Reply:
yes, will fix.